SofascoreAcademy/SofascoreAcademy/Assets.xcassets/loading.imageset/Contents.json

```diff
@@ -1,7 +1,7 @@
 {
   "images" : [
     {
-      "filename" : "nba.png",
+      "filename" : "loading.jpeg",
```

Comment:
Jel ti treba ovaj loading? Jel se nije mogao korisitit UIActiviryIndicator?

Reply:
Taj loading image je placeholder za slike dok/ako se ne ucitaju, mislila sam da za to ne koristimo activity indicator nego samo neku standin sliku

Reply:
Aha, za placeholder za sliku se inače ne stavlja loading bar, nego ovisno o entiteu. Za npr. team se stavi neki blank grb, za playera blank user profile image...